src/templates/base.js

```diff
@@ -123,3 +119,20 @@ ${
 Made with [createapp.dev](https://createapp.dev/)
 
 `;
+
+export const gitignore = () => `
+.cache/
+coverage/
+dist/
```

Comment:
currently, the `index.html` file is placed inside `dist` folder. Maybe we should consider changing that. But in the meantime, let's not add `dist/` to `.gitignore`

Reply:
@jakoblind 
I can see 2 "more proper" options:

1. add `!dist/index.html` right after `dist/` into the `.gitignore` file;
1. use [this webpack plugin](https://github.com/jantimon/html-webpack-plugin) to omit keeping any source/versioned files within `dist` directory; it should be safely removable

Reply:
I was thinking to add `html-webpack-plugin` as a selectable feature. Until then let's go for your option 1. That's nice and explicit.

Reply:
@jakoblind Added.